webpack/assets/javascripts/react_app/components/common/Slot/SlotSelectors.js

```diff
@@ -0,0 +1,12 @@
+import { orderBy } from 'lodash';
+import { getComponentsBySlotId } from '../../../../services/ExtendableRegistry';
+
+export const selectComponentByWeight = slotId =>
+  orderBy(getComponentsBySlotId(slotId), ['weight'], ['desc']).map(
```

Comment:
Could you use [sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) instead of using `lodash`?